需要切换依赖的 uuid 包

```
$ go1.12.5 run monitor.go 
# github.com/jdcloud-api/jdcloud-sdk-go/core
../../../go/pkg/mod/github.com/jdcloud-api/jdcloud-sdk-go@v1.9.0/core/Signer.go:170:11: assignment mismatch: 2 variables but uuid.NewV4 returns 1 values
```

原因:
1. go 1.11之后推荐使用 go mod 来管理你的包，go mod 会优先寻找稳定版，即 tag 好的版本
2. go.uuid 包的稳定版本和master版本接口不一致
3. 建议切换到 https://github.com/gofrs/uuid

--- a/README.md
+++ b/README.md
@@ -15,7 +15,7 @@
 1.	京东云Go SDK的下载地址：https://github.com/jdcloud-api/jdcloud-sdk-go 。
 2.	您也可以使用以下命令获取安装包，代码会被下载到GOPATH环境变量中第一个路径src目录中。
 
-    `go get github.com/jdcloud-api/jdcloud-sdk-go/core github.com/satori/go.uuid`
+    `go get github.com/jdcloud-api/jdcloud-sdk-go/core github.com/gofrs/uuid`
 
 ## 调用SDK
 ### 业务侧SDK的调用主要分为4步：

--- a/core/Signer.go
+++ b/core/Signer.go
@@ -28,7 +28,7 @@ import (
 	"strings"
 	"time"
 	"bytes"
-	"github.com/satori/go.uuid"
+	"github.com/gofrs/uuid"
 )
 
 const (

